src/main/java/core/basesyntax/strategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction fruitTransaction) {
+        String fruit = fruitTransaction.getFruit();
+        int quantity = fruitTransaction.getQuantity();
+        Storage.fruitStorage.put(fruit, quantity);
+    }
```

Comment:
The class `BalanceOperationHandler` should be placed into the `strategy` package according to the checklist. However, it's already in the correct package, so please ensure that the package structure in the project is maintained properly.